tests/mir-opt/tail_expr_drop_order_unwind.rs

```diff
@@ -0,0 +1,36 @@
+// skip-filecheck
+// EMIT_MIR_FOR_EACH_PANIC_STRATEGY
+// EMIT_MIR tail_expr_drop_order_unwind.method_1.ElaborateDrops.after.mir
+
+#![deny(tail_expr_drop_order)]
+
+use std::backtrace::Backtrace;
```

Comment:
This was probably unintentionally copy-pasted from [my mcve](https://github.com/rust-lang/rust/issues/134482)

Reply:
lol ya, doesn't really matter bc i'm not gonna fix it now, and doesn't affect the test